### PR TITLE
gsearch win10 data 20170505

### PR DESCRIPTION
**Pull request by ypwalter**

### CODE REVIEW

**Comment by ShakoHo** [comment]
@MikeLien

**Comment by ShakoHo** [comment]
lgtm r+

File: 20170504/gsearch/win10/gsearch_ail_select_image_cat.json [hunk 1]
```diff
@@ -0,0 +1,822 @@
...
+    "time_list": [
+      {
+        "folder": "test_chrome_gsearch_ail_select_image_cat_1493888430", 
+        "time_sequence": [], 
+        "run_time": 155.55555555555475
+      }, 
+      {
+        "folder": "test_chrome_gsearch_ail_select_image_cat_1493889937", 
+        "time_sequence": [], 
+        "run_time": 766.6666666666679
```
Comment (MikeLien) [inline 20170504/gsearch/win10/gsearch_ail_select_image_cat.json:423]: This should be an outlier, please verify it if correct or not before we merge.

File: 20170504/gsearch/win10/gsearch_ail_type_searchbox.json [hunk 2]
```diff
@@ -0,0 +1,822 @@
+{
+  "test_chrome_gsearch_ail_type_searchbox": {
+    "total_time": 1955.5555555555547, 
+    "time_list": [
+      {
+        "folder": "test_chrome_gsearch_ail_type_searchbox_1493894998", 
+        "time_sequence": [], 
+        "run_time": 200.0
```
Comment (MikeLien) [inline 20170504/gsearch/win10/gsearch_ail_type_searchbox.json:8]: This should be an outlier, please verify it if correct or not before we merge.

File: 20170504/gsearch/win10/gsearch_ail_type_searchbox.json [hunk 3]
```diff
@@ -0,0 +1,822 @@
...
+    "time_list": [
+      {
+        "folder": "test_firefox_gsearch_ail_type_searchbox_1493902348", 
+        "time_sequence": [], 
+        "run_time": 22.22222222221899
+      }, 
+      {
+        "folder": "test_firefox_gsearch_ail_type_searchbox_1493902603", 
+        "time_sequence": [], 
+        "run_time": 166.66666666667152
```
Comment (MikeLien) [inline 20170504/gsearch/win10/gsearch_ail_type_searchbox.json:423]: This should be an outlier, please verify it if correct or not before we merge.

File: 20170504/gsearch/win10/gsearch_ail_type_searchbox.json [hunk 4]
```diff
@@ -0,0 +1,822 @@
...
+      }, 
+      {
+        "folder": "test_firefox_gsearch_ail_type_searchbox_1493909289", 
+        "time_sequence": [], 
+        "run_time": 22.22222222221899
+      }, 
+      {
+        "folder": "test_firefox_gsearch_ail_type_searchbox_1493909558", 
+        "time_sequence": [], 
+        "run_time": 155.55555555555475
```
Comment (MikeLien) [inline 20170504/gsearch/win10/gsearch_ail_type_searchbox.json:553]: This should be an outlier, please verify it if correct or not before we merge.

File: 20170504/gsearch/win10/gsearch_ail_type_searchbox.json [hunk 5]
```diff
@@ -0,0 +1,822 @@
...
+      }, 
+      {
+        "folder": "test_firefox_gsearch_ail_type_searchbox_1493908209", 
+        "time_sequence": [], 
+        "run_time": 22.222222222226264
+      }, 
+      {
+        "folder": "test_firefox_gsearch_ail_type_searchbox_1493908476", 
+        "time_sequence": [], 
+        "run_time": 133.33333333332848
```
Comment (MikeLien) [inline 20170504/gsearch/win10/gsearch_ail_type_searchbox.json:533]: This should be an outlier, please verify it if correct or not before we merge.

**Review by MikeLien — COMMENTED** [review]
please verify some possible outliers are correct or not before we merge them

**Pull request merged**